src/proc_image_processing/cpu/config.h

```diff
@@ -9,12 +9,15 @@ namespace proc_image_processing {
 
     const std::string kProjectPath = std::string(std::getenv("NODE_PATH"));
 
-    const std::string kConfigPath = kProjectPath + "/config";
+    const std::string kConfigPath = kProjectPath + std::string(
```

Comment:
Good job on checking getenv() result! (else the constructor of the string throw an error, not always clear)

Reply:
Actually, it's to support using a different path for the tests